pulsar-client/src/main/java/org/apache/pulsar/client/impl/transaction/TransactionCoordinatorClientImpl.java

```diff
@@ -113,12 +113,8 @@ public CompletableFuture<Void> startAsync() {
     }
 
     private String getTCAssignTopicName(int partition) {
```

Comment:
Maybe we can use `SystemTopicNames.TRANSACTION_COORDINATOR_ASSIGN + TopicName.PARTITIONED_TOPIC_SUFFIX + partition;` directly in line 93 to avoid adding an extra stack frame.